modules/ROOT/pages/_partials/sync/js-client-realtime-updates.adoc

```diff
@@ -1,3 +1,77 @@
 = Realtime Updates
 
-TODO
\ No newline at end of file
+Subscriptions are a core concept of GraphQL and one which allow us to enable real time updates of data
+from the server to the client.
+To do this GraphQL Subscriptions utilise websockets to enable clients to subscribe to published changes.
+
+The architecture of websockets is as follows:
+
+* Client connects to websocket server
+* Upon certain events, the server can publish the results of these events to the websocket
+* Any _currently connected_ client to that websocket will receive these results
+* The client can close the connection at any time and will no longer recieve updates
+
+Websockets are not however, a queueing system. For example if the client is offline when the server publish the changes
+the client will not receive them. To receive updates the client must be currently connected to the websocket server,
+updates made over this websocket while the client is offline will not be replayed.
+{sync-client} comes with subscription support out of the box including auto-reconnection upon device restart or network reconnect.
+To enable subscriptions on your client set the following
+paramater in the {sync-client} config object. A DataSyncConfig interface is also available from {sync-client} if you wish to use it.
+
+== Setting up a Client to use Subscriptions
+
+To set up a client to use subscriptions simply provide a wsUrl in your `mobile-services.json` file, {sync-client} will automatically
+apply the necessary logic to handle websocket requests. This can also be provided manually as seen below.
```

Comment:
actually, just look at the showcase app and I don't think this is the case. It looks like developer will need to pass the config to the sync client. See https://github.com/aerogear/ionic-showcase/blob/master/src/app/services/voyager.service.ts#L72